scripts/validate-for-appstore.sh

```
@@ -0,0 +1,338 @@
+  -destination 'platform=iOS Simulator,name=iPhone 15 Pro' \
+  -configuration Release \
+  -skipPackagePluginValidation \
+  CODE_SIGNING_ALLOWED=NO"
+
+if $USE_XCBEAUTIFY; then
+  eval "$BUILD_CMD 2>&1 | xcbeautify"
+else
+  eval "$BUILD_CMD 2>&1 | grep -E '(error:|warning:|Build Succeeded|BUILD SUCCEEDED)'" || true
+fi
+
+if [ ${PIPESTATUS[0]} -eq 0 ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)